release-content/0.15/release-notes/14920_Add_cached_run_system_API.md

```diff
@@ -1,4 +1,28 @@
-<!-- Add cached `run_system` API -->
-<!-- https://github.com/bevyengine/bevy/pull/14920 -->
+# Cached one-shot systems
 
-<!-- TODO -->
+**Bevy 0.15** introduces a new "cached" API for running one-shot systems:
+
+```rust
+// Run a system without manually registering it first:
+commands.run_system_cached(foo);
+commands.run_system_cached_with(bar, 42);
```

Comment:
```suggestion
// You can pass input into one-shot systems!
commands.run_system_cached_with(bar, 42);
```

Reply:
This might read like passing inputs is a new feature. It does help explain this line though, since passing input into a system is less common. Maybe the `run_system_cached_with` line should just be removed?

Reply:
Yeah, let's cut it.